packages/dev/addons/src/htmlMesh/htmlMeshRenderer.ts

```diff
@@ -297,7 +297,7 @@ export class HtmlMeshRenderer {
     }
 
     // prettier-ignore
-    protected _getCameraCSSMatrix(matrix: Matrix): string {
+    protected _getCameraCssMatrix(matrix: Matrix): string {

```

Comment:
Our coding standards doc (not public) says to use all caps for initialisms. Can linting even tell the difference between initialisms and abbreviations? Maybe we need to modify the doc.

Reply:
Abbreviations are an interesting topic that I will be happy to discuss. We do support abbreviations, but they need to be manually added to the lint file. Let's talk about it tomorrow.

Reply:
I don't know what new docs you mean, but abbreviations are not the same as initialisms.

Reply:
You are right, but that was what it meant. we can discuss this. Anyhow, abbreviations are supported (and configured). You can see it in the linting file (the first line). 
StringPascalCase for example, does not support abbreviations. so It should be `HtmlCssSupportId`. However, with known and supported abbreviations (or initialisms :-) ) we fallback to CamelCase